dev-qt/qtchooser/files/qtchooser-66-qt-5.15.3-compat.patch

```diff
@@ -0,0 +1,33 @@
+--- a/Makefile
++++ b/Makefile
+@@ -1,18 +1,14 @@
+ MKDIR  = mkdir -p
+ prefix = /usr
+ bindir = $(prefix)/bin
+-TOOLS = assistant \
+-	designer \
++TOOLS = designer \
+ 	lconvert \
+-	linguist \
+ 	lrelease \
+ 	lupdate \
+ 	moc \
+ 	pixeltool \
+ 	qcollectiongenerator \
+-	qdbus \
+ 	qdbuscpp2xml \
+-	qdbusviewer \
+ 	qdbusxml2cpp \
+ 	qdoc \
+ 	qdoc3 \
+@@ -44,9 +40,7 @@
+ 	rcc \
+ 	repc \
+ 	uic \
+-	uic3 \
+-	xmlpatterns \
+-	xmlpatternsvalidator \
++	uic3
```

Comment:
would it make sense to also drop the obsolete symlinks while at it, such as uic3, qml1plugindump, etc.? I can get a list if you think it's worthwhile

Reply:
iirc I made some research for those binaries as for when they got obsolete, and I didn't want to drop them for at least Qt4 users (maybe even Qt3 but I have no idea if qtchooser was even relevant for them, and one could argue someone interested in that should use TQt3 instead these days).

Reply:
yeah, makes sense. I was concerned about collisions with Qt6 binaries in `/usr/bin` (such as `qmljs`) but I guess those will have a "6" suffix, so ignore me.